packages/office-ui-fabric-react/src/components/Button/ActionButton/ActionButton.styles.ts

```diff
@@ -15,7 +15,7 @@ export const getStyles = memoizeFunction(
         height: DEFAULT_BUTTON_HEIGHT,
         color: theme.palette.neutralPrimary,
         backgroundColor: 'transparent',
-        border: 'none'
+        border: '1px solid transparent'
```

Comment:
@micahgodbolt is right in his comment in the #7528 when he mentions the possible breaking changes this change might bring.
https://codepen.io/vitalius1/pen/maLqOW in this codepen I do the same thing as you did through the styles prop and if you try to comment it in and out you can see how it shifts the button to right by 1px. This potentially might break some layouts in case there are multiple ActionButtons in one line.